cmd/podman/root.go

```diff
@@ -289,13 +290,19 @@ func rootFlags(cmd *cobra.Command, opts *entities.PodmanConfig) {
 	}
 }
 
-func resolveDestination() (string, string) {
-	if uri, found := os.LookupEnv("CONTAINER_HOST"); found {
```

Comment:
Shouldn't we continue supporting the old variables too?

Reply:
Environment variables CONTAINER_* support comes via the cfg.ActiveDestination() call.  The code here existed before that refactor and was duplicaticated.